about.php

```diff
@@ -20,28 +20,23 @@
 	</div>
 	<div class="heritage">
 		<h3>Our Heritage</h3>
-		<p>We are humbled by the success of our company thus far. The name Starbulls was created by one of the founders:
-			Will. He came up with this insane concept of a restaurant that serves wings and coffee.
-		<p>
+		<p>We are humbled by the success of our company thus far. The name StarBulls was created by Will, 
+			one of our founders. Fueled by a continual flow of caffeine and blue cheese, Will opened StarBulls in Febuary of 2021.
+		</p>
 		<p>How do we measure success? Simply: we started from the bottom, now we are here! Seriously,
-			look at our site! A beauty to say the least.</p>
+			look at our site! A beauty to say the least.
+		</p>
 		<p>Just a week ago, StarBulls did not exist. It has taken hours of dedication to get this
-			company up
-			on the server. Hours of frustration, trying to figure out why the code looks different on
-			our
-			screens. Hours of trial and error as we figured out where we went wrong. One thing the
-			founders
-			of StarBulls has known from the beginning is: Quitters Never Win. And Winners Never Quit. At
-			StarBulls we are
-			Winners.
+			company up on the server. Hours of frustration, trying to figure out why the code looks different on
+			our screens. Hours of trial and error as we figured out where we went wrong. One thing the
+			founders of StarBulls knew from the beginning is: Quitters Never Win. And Winners Never Quit. At
+			StarBulls we are Winners!
 		</p>
 	</div>
 	<div class="tools">
 		<h3>Tools We Used</h3>
-		<p>Like any other company, the founders of StarBulls relied on various resources and tools that helped
-			them
-			get the site up and running! We would like to share these resources/tools with the
-			world.
+		<p>Like any other company, the founders of StarBulls rely on various resources and tools to help
+			create this site. We want to share our tools and resources with the world!
 		</p>
 		<h3>They are:</h3>
 		<ol>
```

Comment:
Wording Suggestion Line 52:
relied on one another and our fellow classmates.

Reply:
um my line 52 is `</button>`. Am I missing something?

Reply:
Also @vricci518 are you clicking add comment or start review? You should be using the start review button. I only ask because the format seems off, though GH may have made an update.

Reply:
I thought these comments would not show until I made a review. Learning moment.

Reply:
![image](https://user-images.githubusercontent.com/888231/112181265-a5717000-8bd2-11eb-82ca-495ec0be24f4.png)

just in case

Reply:
@vricci518 & @billythakidd04 Instead of including classmates on this line, what are your thoughts on including a meet the team page where we could put a mini bio for every class member that then links out to everyone's individual projects? Definitely heading out of scope with this one but I think it could be cool

Reply:
I think its a great idea. Make a ticket. Def not on this one though.

Reply:
resolve?